.ci/scripts/metricbeat-test.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
Do you prefer a script (mainly used by the CI) or a Make goal in the e2e directory, so that developers can run them locally? There are already a few examples for fleet in that Makefile.

Reply:
Will the retry in the CI something that could be also included in the make goal? See https://github.com/elastic/e2e-testing/pull/705/files#diff-12ed1626622bd381afb059882db321d0014a9e9e394c4ca6ff4731887d912076R20-R28

No strong opinion whether to use a sh or a make goal as long as it's just a simple call that the consumer of this project needs to care about

Reply:
Mmm that's right: I did not look at the retry command. Well, if it's simple as it is, then I'm OK with it.

BTW, this script is needed because we need to install the dependencies, and it's done in the Jenkinsfile, right? Wdyt about simplifying the Jenkinsfile to execute the same as in the consumer, with one script? It will simplify the Jenkinsfile (less calls) and the future script will be self-contained, following same strategy as the script in this PR.

Reply:
> BTW, this script is needed because we need to install the dependencies, and it's done in the Jenkinsfile, right?

Yes

> Wdyt about simplifying the Jenkinsfile to execute the same as in the consumer, with one script? 

I was tempted to do so, but it might require further surgery, let me give a go

Reply:
Let me know if we can collaborate there

Reply:
Done, I restructure the scripts so this is now a dummy one to call the functional-test.sh with some parameters, we could even potentially remove it, though I like the idea of using this contract